dockerfiles/haskell/3.0.0.0/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:22.04
+
+# Install necessary packages for GHC (compiler)
+RUN apt-get update \
+    && apt-get -y --no-install-recommends install \
+    build-essential \
+    curl \
+    libgmp-dev \
```

Comment:
Are these 3 necessary for install of ghc and cabal-install? Will apt fetch their deps for them?

Reply:
those were needed for `RUN curl -sSL https://get.haskellstack.org/ | sh` but we don't need them, I'll update them.